proteus/mprans/MoveMesh.py

```diff
@@ -493,7 +493,8 @@ def __init__(self,
         compKernelFlag = 0
         if self.nSpace_global == 2:
             import copy
-            self.u[2] = copy.deepcopy(self.u[1])
+            self.u[2] = self.u[1].copy()
```

Comment:
Some of the issues I encountered with the MeshAdapt workflow in this PR stemmed from the switch to shallow copies from deep copies. In my case, the deep copies were needed to maintain a history of values, but a deepcopy wasn't doable with the changes from this PR because of some error like `self.mesh object could not be pickled`. Is that the reasoning for this line of code? And are we sure that there is a test that accounts for this?

Reply:
I see. Thanks for the clarification. The custom copy should be a safe switch since it's based off the `numpy` copy routine, so I no longer think it's necessary to confirm that all lines are covered by tests.
Apart from the `deepcopy` issues that I mentioned in the adaptivity workflow (which I fixed with several commits), I'm not seeing any additional errors. 

I think this is good to go.